environment-dev.yml

```diff
@@ -2,54 +2,43 @@ name: pyjanitor-dev
 channels:
   - conda-forge
 dependencies:
-  - python=3.9
+  - python>=3.6
```

Comment:
Now minimal python version is 3.6
It may don't work in py3.6 with pandas 1.4.1.
it will be updated after setting `ci/env` multi-envs

Reply:
is python 3.6 still active? why not 3.7

Reply:
py3.6 base on setup.py.
As you said, this is the reason I want to build multi-envs
we could test pyjanitor compatible with python3.6 after `ci/env/` was built.
https://github.com/pyjanitor-devs/pyjanitor/blob/1908f353051d22c6fadc4675ed475b9e273b61eb/setup.py#L124

Reply:
@samukweku @Zeroto521 thanks for doing the PR and reviewing!

I think @samukweku's suggestion makes sense here. With `pandas` minimum version being 3.8, we should just follow along. I think `setup.py` should be updated.